### DISCUSSION TITLE
how to set password?

### DISCUSSION THREAD

**Discussion by zenghaojim33** [spencerwooo/onedrive-vercel-index#213] [Q&A] [answered]
According to documentation, password is store in `.password` file in plain text (is it?), but it doesn't seem to work - what's the proper way to do the setting?

**Comment by spencerwooo** [comment, marked as answer]
1. `.password` store plain text password, regardless of format
2. the route that holds the `.password` file, i.e., the folder you are trying to protect, must be defined in `config/site.json` under `protectedRoutes`

https://github.com/spencerwooo/onedrive-vercel-index#-password-protected-folders
